scripts/about_taskw-android_fork.md

```diff
@@ -0,0 +1,6 @@
+### About the [timew-android](https://github.com/linuxcaffe/timew-android) fork;
+It is intended to adapt [ATimeTracker](https://github.com/netmackan/ATimeTracker) as an android front-end for [timewarrior](https://timewarrior.net/), first, by providing script(s) to convert exported csv to timewarrior data, eventually, by converting the back-end to timewarrior-native data.
+
+.. *but timewarrior is a linux cli script* you say? what does that have to do with my android phone?
+
+Well timewarrior is one of may packages available to [termux](https://termux.com/) (a linux shell application for android) and timew can be easily installed and run in android. With this in mind, these scripts are intended to operate on an android phone with termux, timew and atimetracker installed. 
```

Comment:
Spelling: "is one of may packages". I guess it should be "one of many packages" ?